.github/scripts/merge.py

```diff
@@ -0,0 +1,88 @@
+import os
+import shutil
+import distutils.dir_util
+import distutils.file_util
+
+
+# copy all code from bot except readme
+def copy_bot_folder(botfolder, mefolder, combinedfolder):
+    distutils.dir_util.copy_tree(botfolder, combinedfolder)
+    distutils.file_util.copy_file(os.path.join(mefolder, "README.md"), combinedfolder)
+    distutils.dir_util.copy_tree(os.path.join(mefolder, "images"), os.path.join(combinedfolder, "images"))
+
+
+# merge pkg reference
+def merge_pkg_reference(bot_lines, me_lines, merged_lines):
+    for line in me_lines:
+        if not line.startswith(("import", "const")):
+            break
+        if "botbuilder" not in line and line not in bot_lines:
+            merged_lines.insert(0, line)
+
+
+# merge functions in TeamsActivityHandler class
+def merge_functions_in_class(bot_lines, me_lines, merged_lines):
```

Comment:
NIT: the solution looks like an ad-hoc string analyze, it highly depends on some implicit code constraints. In fact, my prefer solution is using a template engine (like jinja since you use python, or mustache which we are using now) to generate code based on parameters. So user who know the template engine can easily review your templates.

Reply:
Good suggestion, I will consider this solution